mediaQueries.css

```diff
@@ -0,0 +1,144 @@
+@media only screen and (min-width: 786px) {
+  header {
+    position: absolute;
+    width: 100%;
+    height: 44px;
+    left: 1px;
+    top: 1px;
+  }
+
+  #menu {
+    display: none;
+  }
+
+  .navbar ul li {
+    display: inline;
+  }
+
+  .text-holder {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    padding-top: 100px;
+    width: 500px;
+  }
+
+  .works-section {
+    margin-top: 100px;
+    display: grid;
+    align-items: center;
+    justify-content: center;
+    grid-template-columns: 700px;
+    grid-template-rows: repeat(4, 596px);
+    grid-column-gap: 10px;
+    grid-row-gap: 30px;
+  }
+
+  .card {
+    display: flex;
+    flex-direction: row;
+    height: 450px;
+    width: 700px;
+    background-color: rgb(255, 255, 255);
+    border-radius: 15px;
+    padding: 24px;
+  }
+
+  .card-1 {
+    grid-row-start: 3;
+    grid-row-end: 4;
+  }
+
+  .card-3 {
+    grid-row-start: 4;
+    grid-row-end: 5;
+  }
+
+  .card-img-3 {
+    order: 2;
+  }
+
+  .card-4 {
+    grid-row-start: 2;
+    grid-row-end: 3;
+  }
+
+  .card-img-4 {
+    order: 2;
+  }
+
+  .card-img {
+    margin-top: 20px;
+    width: 270px;
+    height: 350px;
+  }
+
+  .sub-item {
+    margin-top: 20px;
+  }
+
+  .card-text {
+    margin-top: 32px;
+  }
+
+  .tech-used {
+    margin-top: 28px;
+  }
+
+  .see-project {
+    margin-top: 52px;
+  }
+
+  .about-myself {
+    height: 924px;
+    flex-direction: row;
+    align-items: flex-start;
+    justify-content: center;
+    padding-top: 166px;
+  }
+
+  .dropdown-item {
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    width: 122px;
+    height: 120px;
+  }
+
+  .details-2-title {
+    margin: 10px;
+  }
+
+  .contact {
+    height: 779px;
+    width: 100%;
+    margin-top: -140px;
+    display: flex;
+    justify-content: space-around;
+    flex-direction: column;
+    text-align: center;
+    background-size: cover;
+    background-image: url("./img/contact.png");
+    background-color: rgb(96, 112, 255);
+    align-items: center;
+    padding: 100px 24px;
+    border-top-left-radius: 100px;
+  }
```

Comment:
- [ ] Right now your portfolio should be responsive on all devices but there a problem with tablets devices  please try to fix that 

- [ ]  Also please try to use the right illustration background for the desktop version as you can see the illustration is not clean in the desktop version

![image](https://user-images.githubusercontent.com/61790587/177780366-d63b889f-5335-4908-b7cb-ca3f26320df7.png)